boards/shields/nrf2220ek/boards/nrf5340dk_nrf5340_cpunet.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_BOOT_BANNER=n
+CONFIG_EARLY_CONSOLE=n
+CONFIG_NCS_BOOT_BANNER=n
```

Comment:
as above, unless this is because the UART is unusable because of SPI being used? If so, UART should be disabled in dts overlay for this board

Reply:
This code has been just moved from old:
https://github.com/nrfconnect/sdk-nrf/blob/f4100a58a8fcfb9daf111c31cc1e10e53565fad5/boards/shields/nrf2220ek/nrf2220ek.conf

Reply:
This PR is just to make build simpler for an user, to provide just one -DSHIELD. It doesn't attempt to fix all and pab back technical debt that is taken.

Please have a look https://github.com/nrfconnect/sdk-nrf/blob/f4100a58a8fcfb9daf111c31cc1e10e53565fad5/subsys/mpsl/fem/nrf2220/mpsl_fem_nrf2220.c#L37

It has been so not by coincidence (however I don't remember now why all these options must be turned off). This is technical debt for a not-yet-released nRF2220 and nRF2240 FEMs that we at RS team would like to solve later.
Would you accept paying back the technical debt later?

Reply:
@nordicjm this issue is addressed in PR https://github.com/nrfconnect/sdk-nrf/pull/19057
Once that is merged I'll rebase and edit commits in this PR here and will have the options you pointed removed.

Reply:
@nordicjm addressed this issue, rebased, please have a look again